app/src/main/res/layout/help_without_drawer_activity.xml

```diff
@@ -8,9 +9,15 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <include
-      android:id="@+id/help_activity_toolbar"
-      layout="@layout/toolbar" />
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/help_activity_toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/component_color_shared_activity_toolbar_color"
+        android:minHeight="?attr/actionBarSize"
+        app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
+        app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
+        app:titleTextAppearance="@style/ToolbarTitle"  />
```

Comment:
Nit: extra space before the ``/``.

Reply:
That extra space has now been removed, with the latest push.